api/v1alpha1/twemproxy_types.go

```diff
@@ -0,0 +1,108 @@
+package v1alpha1
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+var (
+	// Twemproxy defaults
+	// WARNING -> REVIEW THIS!!!
```

Comment:
Is this intended (`Warning`, here and on next lines)?

Reply:
Oh, I already reviewed this,and per the testing seems to work fine. I just forgot to delete the comment. Fixed with 0174f7a.